### PR TITLE
Custom Enumerable Module

### PR DESCRIPTION
**Pull request by Gegardus**
In this project, we completed the following tasks
- [ ] Created a class `MyList` that has an instance variable `@list`.
- [ ] Implemented in `MyList`  a method` #each` that yields successive members of `@list` and uses the `MyEnumerable` module.
- [ ] Created a module `MyEnumerable` that implements the following methods (with the same functionality as methods in Enumerable):
   - #all?
   - #any?
   - #filter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+- Ruby
+
+## Getting Started
+
+To get the content of this project locally you need to run these commands in your terminal.
+Before starting the steps below, be sure you have [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/).
+1. Download the `Zip` or clone with `git clone https://github.com/Gegardus/custom-enumerable`
+2. `cd Decode-Morse-Message-Ruby`
+
```
Comment (julie-ify) [inline README.md]: - [x] Kindly update `cd Decode-Morse-Message-Ruby` to the current project 😄

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Gegardus and @tresorsawasawa,

Good job so far!

## Highlights! 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gegardus** [comment]
Thank you and sorry for such an error.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+## Built With
+
+- Ruby
+
+## Getting Started
+
+To get the content of this project locally you need to run these commands in your terminal.
+Before starting the steps below, be sure you have [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/).
+1. Download the `Zip` or clone with `git clone https://github.com/Gegardus/custom-enumerable`
+2. `cd custom-enumerable`
```
Comment (Dipeshtwis) [inline README.md:16]: [OPTIONAL]

- In the README file, please mention the step on how to test the application. Hint: `ruby my_list.rb`.

**Review by Dipeshtwis — APPROVED** [review]
# Hi Team, 

### Status: Approved ✔️

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Gegardus) [reply README.md:16]: Thank you @Dipeshtwis for your approval and suggestion.

**Pull request merged**